upstream https://github.com/PeteDavidson/t38modem/pull/1
commit 087e49a0ed6e8c99ea1582ed1f65e42e60fd26d4
Author: Jan Engelhardt <jengelh@inai.de>
Date:   Tue Sep 1 21:36:54 2015 +0200

    build: resolve undefined operation
    
    gcc warns:
    
    pmodeme.cxx: In member function 'int ModemEngineBody::NextSeq()':
    pmodeme.cxx:940:54: warning: operation on
    '((ModemEngineBody*)this)->ModemEngineBody::seq' may be undefined
    [-Wsequence-point]
         int NextSeq() { return seq = ++seq & EngineBase::cbpUserDataMask; }

diff --git a/pmodeme.cxx b/pmodeme.cxx
index f0babb2..1c3ba86 100644
--- a/pmodeme.cxx
+++ b/pmodeme.cxx
@@ -937,7 +937,7 @@ class ModemEngineBody : public PObject
     void _DetachEngine(ModemClassEngine mce);
     void _ClearCall();
 
-    int NextSeq() { return seq = ++seq & EngineBase::cbpUserDataMask; }
+    int NextSeq() { ++seq; return seq &= EngineBase::cbpUserDataMask; }
 
     ModemEngine &parent;
 
